sos/report/plugins/kdump.py

```diff
@@ -127,6 +144,22 @@ def setup(self):
             "/etc/default/kdump-tools"
         ])
 
+        try:
+            path = self.read_kdump_conffile()
+        except Exception:  # pylint: disable=broad-except
+            # set default path of coredir
+            path = "/var/crash"
+
+        self.add_dir_listing(path, recursive=True)
+        self.add_copy_spec(f"{path}/kexec_cmd")
+        self.add_copy_spec(f"{path}/kdump_lock")
```

Comment:
Why do we need to capture this file ? AFAIK this only indicates that a kdump is present in the directory

Reply:
@vaibhav92 `kexec_cmd` and `kdump_lock` might be needed for to debug kdump issue itself.